### PR TITLE
Feat/add pymechanical

### PR DESCRIPTION
**Pull request by rs-bh-n**
added pymechanical cheat sheet

### CODE REVIEW

**Comment by Revathyvenugopal162** [comment]
Thank you @rs-bh-n for opening the PR, please provide the screenshot/pdf file of the PDF generated  in the description of the PR, so that it will be easy for docs team for review. Adding @EstherProsperi for the proof reading.

**Comment by rs-bh-n** [comment]
Hi @Revathyvenugopal162 and @EstherProsperi  , here is the snapshot of the pdf generated.

image

**Comment by EstherProsperi** [comment]
> Hi @Revathyvenugopal162 and @EstherProsperi , here is the snapshot of the pdf generated.
> 
> image

Any chance I could have the actual PDF?

**Comment by rs-bh-n** [comment]
[pymechanical_cheat_sheet.pdf](https://github.com/ansys/pyansys-cheat-sheet/files/12092267/pymechanical_cheat_sheet.pdf)

Here it is , @EstherProsperi .

**Comment by EstherProsperi** [comment]
> [pymechanical_cheat_sheet.pdf](https://github.com/ansys/pyansys-cheat-sheet/files/12092267/pymechanical_cheat_sheet.pdf)
> 
> Here it is , @EstherProsperi .

[pymechanical_cheat_sheet.pdf](https://github.com/ansys/pyansys-cheat-sheet/files/12093755/pymechanical_cheat_sheet.pdf)

**Comment by Revathyvenugopal162** [comment]
@rs-bh-n can you please adapt the review from Ester? if you encounter any issue/ roadblock, let me know i will try to adapt Thanks.

File: makefile [hunk 1]
```diff
@@ -1,7 +1,6 @@
 BUILD_DIR     = _build
 
-CHEATSHEETS   = pymapdl_cheat_sheet pyfluent_cheat_sheet pyaedt_API_cheat_sheet pyedb_API_cheat_sheet pyprimemesh_cheat_sheet pydpf-core_cheat_sheet pydynamicreporting_cheat_sheet
-
+CHEATSHEETS   = pymapdl_cheat_sheet pyfluent_cheat_sheet pyaedt_API_cheat_sheet pyedb_API_cheat_sheet pyprimemesh_cheat_sheet pydpf-core_cheat_sheet pymechanical_cheat_sheet pydynamicreporting_cheat_sheet
```
Comment (Revathyvenugopal162) [inline makefile:3]:
```suggestion
CHEATSHEETS=pymapdl_cheat_sheet pyfluent_cheat_sheet pyaedt_API_cheat_sheet pyedb_API_cheat_sheet pyprimemesh_cheat_sheet pydpf-core_cheat_sheet pymechanical_cheat_sheet pydynamicreporting_cheat_sheet
```

**Comment by rs-bh-n** [comment]
Thanks for your guidance , @Revathyvenugopal162  , in git push.

@EstherProsperi , Thanks for your inputs on wednesday :  Here is the pdf with the below changes . Let me know if this is okay.

[pymechanical_cheat_sheet.pdf](https://github.com/ansys/pyansys-cheat-sheet/files/12122538/pymechanical_cheat_sheet.pdf)


1) Titles changed(same as in docs now. with underscores intact) : 
A.Connect to a Remote Session of Mechanical from Python
B.Load an Embedded instance of Mechanical in Python

2) Retained the title "Launch and Connect to a Session " ( a and b are methods of doing this )

3) Local and Remote : Reworded , comment modified

4) Subtext changes :
	"Example: Import file and print the count of bodies"
	"Example: Import file and print the count of bodies"
	"To turn on logging for warnings:"
	
5)python corrections : added missing import command .added black format .and  set the reference links as two rows.

**Comment by rs-bh-n** [comment]
Thanks @EstherProsperi , for your inputs.
Attached are the pdfs.

@Revathyvenugopal162 , the documentation approval is done.

**Review by Revathyvenugopal162 — APPROVED** [review]
Looks good . Thankyou for the cheatsheet @rs-bh-n and thank you @EstherProsperi for the quick and thorough review,

**Comment by Revathyvenugopal162** [comment]
merging!

**Pull request merged**